CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [Unreleased]
+
+#### Warning - Potentially breaking change
+- Restivus used to store the account login token in the user document: `services.resume.loginTokens.token`
```

Comment:
I think the first line in this warning should be something along the lines of: `All clients consuming a Restivus API will need to reauthenticate after this update, as their existing credentials will be rendered invalid".